compiler/z/codegen/OMRInstOpCode.hpp

```diff
@@ -631,13 +631,11 @@ class InstOpCode: public OMR::InstOpCode
    uint64_t is32bit() {return properties[_mnemonic] & S390OpProp_Is32Bit;}
    uint64_t is32to64bit() {return properties[_mnemonic] & S390OpProp_Is32To64Bit;}
 
-   uint64_t usesImplicitRegisters() { return properties[_mnemonic] & S390OpProp_UsesImplicitRegisters;}
    uint64_t hasLongDispSupport() {return properties[_mnemonic] & S390OpProp_LongDispSupported;}
    uint64_t usesRegPairForTarget() {return properties[_mnemonic] & S390OpProp_UsesRegPairForTarget; }
-   uint64_t mayUseRegPairForTarget() { return properties[_mnemonic] & S390OpProp_MayUseRegPairForTarget; }
    uint64_t usesRegPairForSource() {return properties[_mnemonic] & S390OpProp_UsesRegPairForSource; }
    uint64_t usesRegRangeForTarget(){return properties[_mnemonic] & S390OpProp_UsesRegRangeForTarget; }
-   uint64_t canUseRegPairForTarget() {return usesRegPairForTarget() || usesRegRangeForTarget() || mayUseRegPairForTarget(); }
```

Comment:
`mayUseRegPairForTarget` has been deprecated as it was only set on a single instruction. This instruction was `SIGP` which _may_ set either the first or the second register in the first operand, whichever is odd. To make things simple we enforce a restriction that the first operand must be an even-odd register pair, such that we always know `SIGP` will use the odd register of the respective pair.